### ISSUE TITLE
[Mimir] Add a "simple-scalable" version of helm chart

### ISSUE THREAD

**Issue by kszlachcikowski-locon** [grafana/helm-charts#1189]
Similar to Loki deployment charts add a "simple-scalable" option for Mimir (`--target=full`) where deploying in full microservices mode is too complex/unnecessary.

**Comment by flokli** [comment]
Also very interested in this :-)

**Comment by AlexisDucastel** [comment]
Can't wait to have such chart :) I would be very awesome to provide a step between single monolithic and full micro services.
👍

**Comment by gclawes** [comment]
Would this effectively be a monolithic deployment if it was configured with just 1 replica?

**Comment by sbhrule15** [comment]
@gclawes not necessarily, as you would still need a read pod and a write pod - the goal here is to eliminate the high inter-AZ networking costs on each path, while still allowing for distributed load with separately scalable write and read needs across availability zones for fault-tolerance.

We use Mimir at [Mosaic Learning](https://www.mosaiclearning.com/), and we had some very high inter-AZ networking costs - we were essentially forced to zone lock each path with affinity rules (many thousands of dollars a month).

**Comment by juanchovelezpro** [comment]
any news about the mimir simple scalable helm chart ?

**Comment by rubenvw-ngdata** [comment]
Since the helm chart has been moved to https://github.com/grafana/mimir, probably good to log a ticket for this over there.
